### PR TITLE
Modified chart spacing in bpmn-explorer and human-task explorer

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: components/bpmn/bpmn-explorer-web/src/web/js/actions.js [hunk 1]
```diff
@@ -481,9 +482,10 @@ function selectProcessForAvgTimeDuration(){
                             title: 'Average Time Time Duration in Minutes',
                             titleTextStyle: {color: 'grey'}
                         },
-                        colors: ['#be2d28'],
-                        height: ((data.getNumberOfRows() + 2) * 120) + 200,
-                        chartArea: {top: 10, height: chartAreaHeight}
+                        height: ((data.getNumberOfRows()+2) * 100) + 200,
```
Comment (sinthuja) [inline components/bpmn/bpmn-explorer-web/src/web/js/actions.js]: There should be a space before and after '+' sign.

File: components/bpmn/bpmn-explorer-web/src/web/js/actions.js [hunk 2]
```diff
@@ -538,13 +540,14 @@ function selectProcessForInstanceCount(){
                     var hTitle='Process instance count';
                     var logScaleEnabled = false;
                 }
-                var chartAreaHeight=((data.getNumberOfRows()+2) * 120);
+                var chartAreaHeight=((data.getNumberOfRows()+2) * 100);
```
Comment (sinthuja) [inline components/bpmn/bpmn-explorer-web/src/web/js/actions.js]: Formatting issue. Space after and before + sign.

File: components/bpmn/bpmn-explorer-web/src/web/js/actions.js (hunk 1 above)
Comment (NatashaWso2) [inline components/bpmn/bpmn-explorer-web/src/web/js/actions.js]: Fixed
https://github.com/wso2/carbon-business-process/blob/master/components/bpmn/bpmn-explorer-web/src/web/js/actions.js#L621

File: components/bpmn/bpmn-explorer-web/src/web/js/actions.js (hunk 2 above)
Comment (NatashaWso2) [inline components/bpmn/bpmn-explorer-web/src/web/js/actions.js]: Fixed
https://github.com/wso2/carbon-business-process/blob/master/components/bpmn/bpmn-explorer-web/src/web/js/actions.js#L678

**Pull request merged**